docs/tests.md

```diff
@@ -0,0 +1,18 @@
+# Tests for actors
+
+You can implement two types of tests for an actor in the leapp-repository - functional and unit tests.
+
+## Functional tests
+   - These tests provide fabricated input messages for the actor, check the output messages and check for side effects stated in the actor's description.
+   - These tests should consider the actor to be a black box, so ideally they should written by somebody who doesn't know the code.
```

Comment:
```suggestion
   - These tests should consider the actor to be a black box, so ideally they could be written by somebody who doesn't know the code.
```

Reply:
Even if I would really like this division between test/actor writer, in real world I believe this would not happen. I propose this change to change the meaning to something like "make sure your actor is well described in a way someone else can write the actor".